tests/ast/data/Session.call.test

```diff
@@ -12,4 +12,4 @@ df2 = session.call("my_sproc")
 
```

Comment:
with sprocs working now, can we fix up this test so "my_sproc" gets properly registered here first? (You can do so either in test setup, or here)

Reply:
done!